build/index.js

```diff
@@ -281,6 +281,10 @@ async function main() {
         return 0;
     }
 
+    if (command === "clean") {
+        await npm("run", "clean");
```

Comment:
I was actually wondering if we shouldn't use `cleanPackage` here, but I guess it's not a good fit, trying to remove stuff like `dist` as well. For now I like it hard coded as well :D

Reply:
Mainly because it's also trying to remove build... Which, btw, we could think about changing the name. I was thinking about changing its name to `bauen` or `polyBuild` or anything else in that vein.